about.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>African Movies Festival</title>
+    <link rel="stylesheet" href="./styles/mobile-about.css">
+    <link media="screen and (min-width: 768px)" rel="stylesheet" href="./styles/desktop-about.css">
+</head>
+
+<body>
+    <navbar class="navabar">
+        <div class="nav-header">
+            <ul>
+                <li><img src="./images/social-01.png" alt=""></li>
+                <li><img src="./images/social-02.png" alt=""></li>
+                <li>
+                    <h2>English</h2>
+                </li>
+                <li>
+                    <h2>My page</h2>
+                </li>
+                <li>
+                    <h2>Logout</h2>
+                </li>
+            </ul>
+        </div>
+
+        <div class="desktop-navbar">
+            <div class="desktop-logo">
+                <a href="index.html"><img src="./images/download.jpg" alt=""></a>
+            </div>
+            <div class="navbar-pages">
+                <ul>
+                    <li><a href="">About</a></li>
+                    <li><a href="">Program</a></li>
+                    <li><a href="">Join</a></li>
+                    <li><a href="">Sponsor</a></li>
+                    <li><a href="">News</a></li>
+                    <li class="navbar-button" id='on'>CC Campaign</li>
+                </ul>
+            </div>
+        </div>
+
+        <!-- mobile navbar -->
+
+        <div class="navbar-container">
+
+            <div class="nav-logo">
+                <a href="#"><img class="back-image" src="./images/hamburger.png" onclick=showMenu() id='menu-bar'
+                        alt="menu"></a>
+            </div>
+            <div class='main-menu'>
+                <div id='close'>
+                    <img src="./images/close.png" onclick=close() alt="close">
+                </div>
+
+
+                <ul class="main-links">
+                    <li>
+                        <a href="about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="#program">Program</a>
+                    </li>
+                    <li>
+                        <a href="#join">Join</a>
+                    </li>
+                    <li>
+                        <a href="#sponsor">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="#News">News</a>
+                    </li>
+                    <li>
+                        <a href="#cc campains" class='highlight' id='on'>CC Campaign</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        </div>
+
+
+    </navbar>
+
+
+    <section class="headline">
```

Comment:
- [ ] Your about page heading is a bit different from the original design. Please make sure that your heading section looks almost similar to the original design, you could try to increase the font size for example, and the width, typo, color, and spacing for this part :+1: